searcher/src/main/java/ru/otus/searcher/applicationRunner/CityDataBaseFiller.java

```diff
@@ -0,0 +1,16 @@
+package ru.otus.searcher.applicationRunner;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.ApplicationEventPublisher;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class CityDataBaseFiller {
```

Comment:
where do you use it?

Reply:
the idea was taken from...

[](https://www.techgeeknext.com/spring-boot/spring-boot-event-listener-example)

not used anymore since ContextRefreshedEvent is self-sufficient

del.